src/test/java/wolox/training/models/BookTest.java

```diff
@@ -0,0 +1,196 @@
+package wolox.training.models;
+
+import com.github.javafaker.Faker;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import wolox.training.utils.ValuesGenerator;
+
+/**
+ * Testing for the {@link Book} class.
+ */
+class BookTest {
+
+
+    @Test
+    @DisplayName("Create Book - Valid arguments")
+    void testCreateBookWithValidArguments() {
+        Assertions.assertDoesNotThrow(
+            () -> new Book(
```

Comment:
maybe to no repeat always constructor with all params, you can use a hash map.

Reply:
Done. Check the `ModelsTestHelper` class.